slave.mk

```diff
@@ -36,7 +36,7 @@ PYTHON_WHEELS_PATH = $(TARGET_PATH)/python-wheels
 PROJECT_ROOT = $(shell pwd)
 STRETCH_DEBS_PATH = $(TARGET_PATH)/debs/stretch
 STRETCH_FILES_PATH = $(TARGET_PATH)/files/stretch
-DBG_IMAGE_MARK = -dbg
+DBG_IMAGE_MARK = dbg
```

Comment:
Suggest change name to `DBG_IMAGE_SUFFIX`

Reply:
Wish I could. But I have to use this as prefix for final image, as images have different extensions (unlike dockers, which all ends with .gz) as .bin, .swi, .raw, .... If I make it suffix, it would be something like sonic-broadcom-dbg.bin, sonic-broadcom-dbg.swi. With all Makefile variables having 'sonic-broadcom.bin' as prefix, stripping off dbg from the middle leads to complex rules. Even if you would make 'sonic-broadcom' as prefix, stripping off dbg-<wild chars> is not easy one within Make and it would make Makefile more unreadable.

Next PR will be the one for final image.

Reply:
Based on your description, it's not really ever being used as a prefix; it's always used as a suffix to the filename (not counting what trails after the '.', because those are technically file *extensions*. Unless I'm missing something, it could still be renamed `DBG_IMAGE_SUFFIX`.

Reply:
I guess, I did not make it clear. Let us just wait till the next PR, which is going to use it as "prefix" for reasons, which can be better explained, when you see the PR.

If we still decide to change, we will do it in the next PR. 

BTW, the next PR will follow as soon as this gets approved.